app/models/clinic.rb

```diff
@@ -22,6 +23,7 @@ class Clinic
 
   # Validations
   validates :name, :street_address, :city, :state, :zip, presence: true
+  validates :name, uniqueness: true
```

Comment:
Believe it or not the rubocop styleguide prefers it this way! Going to keep it this way, at least until they change the style guide (which, most likely, they will :( )